src/lib.rs

```diff
@@ -184,6 +187,9 @@ fn build_poi(row: Row) -> Option<Poi> {
     let id = row.get("id");
     let name: String = row.get("name");
     let class: String = row.get("class");
+    let subclass: String = row.get("subclass");
+    let class_sub: String = format!("class_{} subclass_{}", class, subclass);
```

Comment:
Please make clearer that this string format is indexed as `poi_type.name` and we will rely on it to filter places.